data-grid.adoc

```diff
@@ -1,113 +1,119 @@
-// .. Thread pool size: Number of threads that execute write operations to the index. 
-
-// . Index merge: 
-// .. Calibrate by deletes: Whether the number of deleted entries in an index should be taken into account when counting the entries in the segment. Setting `false` will lead to more frequent merges caused by `max-entries`, but will more aggressively merge segments with many deleted documents, improving search performance.
-// .. Factor: Number of segments that are merged at once. With smaller values, merging happens more often, which uses more resources, but the total number of segments will be lower on average, increasing search performance. Larger values (greater than 10) are best for heavy writing scenarios.
-// .. Max entries: Maximum number of entries that an index segment can have before merging. Segments with more than this number of entries are not merged. Smaller values perform better on frequently changing indexes, larger values provide better search performance if the index does not change often.
-// .. Min size: Minimum target size of segments, in MB, for background merges. Segments smaller than this size are merged more aggressively. Setting a value that is too large might result in expensive merge operations, even though they are less frequent.
-// .. Max size: Maximum size of segments, in MB, for background merges. Segments larger than this size are never merged in the background. Settings this to a lower value helps reduce memory requirements and avoids some merging operations at the cost of optimal search speed. This attribute is ignored when forcefully merging an index and `max-forced-size` applies instead.
-// .. Max forced size: maximum size of segments, in MB, for forced merges and overrides the `max-size` attribute. Set this to the same value as `max-size` or lower. However setting the value too low degrades search performance because documents are deleted.
-
-== Authorization:
-Authorization secures your deployment by restricting access to data based on user roles and permissions.
+Index reader:: The index reader provides access to the indexes to perform queries. As the index content changes, Infinispan needs to refresh the reader so that search results are up to date.
+.. Refresh interval: Interval, in milliseconds, to reopen the index reader. By default Infinispan reads the index before each query if the index changed since the last refresh. Configuring with a value larger than zero will make some queries results stale, but query throughput will increase substantially, specially in write heavy scenarios.
```

Comment:
We can ignore points inside Index tuning for now, just explaining 
1. Index Reader
2. Index writer
would work